targets/kodi

```diff
@@ -5,27 +5,19 @@
 if [ "${TARGETNOINSTALL:-c}" = 'c' ]; then
     if [ "$DISTRO" = 'kali' ]; then
         error 99 "Kodi not supported on Kali Linux."
+    elif [ "$DISTRO" = 'debian' ]; then
```

Comment:
I'll play around with it...  When i tried to install it on sid, it wasn't able to connect to X and couldn't start kodi.  I'll see if i can figure out why tonight.